usaspending_api/etl/management/commands/load_usaspending_assistance.py

```diff
@@ -168,16 +168,17 @@ def get_or_create_recipient(self, row):
                 row,
                 mapper=location_mapper_fin_assistance_recipient).location_id,
             "recipient_name": row['recipient_name'],
-            "recipient_unique_id": row['duns_no'],
         }
 
         recipient_type = row.get("recipient_type", ":").split(":")[1].strip()
         recipient_dict.update(self.recipient_flags_by_type(recipient_type))
 
-        le = LegalEntity.objects.filter(
-            recipient_unique_id=row['duns_no']).first()
-        if not le:
-            le = LegalEntity.objects.create(**recipient_dict)
+        le, created = LegalEntity.get_or_create_by_duns(duns=row['duns_no'])
+        if created:
+            # Update from our recipient dictionary
+            for attr, value in recipient_dict.items():
```

Comment:
Gotcha...I initially flagged because the only I saw being explicitly set on new `LegalEntity` objects created by the submission loader is the location (_i.e._ not the name): https://github.com/fedspendingtransparency/usaspending-api/blob/2ad3d7f49baab760a44df67481148a1e0f1312d0/usaspending_api/etl/management/commands/load_submission.py#L652.

Will the name be handled on the submission side because the incoming field name matches the model name?